docs/source/contributing.md

```diff
@@ -171,3 +171,22 @@ Documentation is automatically published to a docsite via [GitHub Actions](https
 
 Portions of this contribution guide were sourced from [pyctyominer](https://github.com/cytomining/pycytominer/blob/master/CONTRIBUTING.md).
 Many thanks go to the developers and contributors of that repository.
+
+## Publishing Releases
+
+### Release Locations
+
+We utilize [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) ("semver") in order to distinguish between major, minor, and patch releases.
+We publish source code by using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) available [here](https://github.com/cytomining/CytoTable/releases).
+We publish Python packages through the [Python Packaging Index (PyPI)](https://pypi.org/) available [here](https://pypi.org/project/cytotable/).
+
+### Publishing Process
+
+There are several manual and automated steps involved with publishing CytoTable releases.
+See below for an overview of how this works.
+
+1. Prepare a release in a code contribution which utilizes the command [`poetry version ...`](https://python-poetry.org/docs/cli/#version) to update the version (this updates `pyproject.toml` with automatically incremented versions under `version = "..."`).
+1. Open a pull request and use a repository label for `release-<semver release type>` to label the pull request for visibility with [`release-drafter`](https://github.com/release-drafter/release-drafter).
```

Comment:
Is there an example PR you can point to?

Reply:
Thanks, great point! I'll add a reference to this PR as an example of this occurring within CytoTable.